apps/vikunja/docker-compose.yml

```
@@ -63,7 +62,7 @@ services:
       # Main
       traefik.enable: true
       traefik.http.middlewares.vikunja-web-redirect.redirectscheme.scheme: https
-      traefik.http.services.vikunja.loadbalancer.server.port: 80
+      traefik.http.services.vikunja.loadbalancer.server.port: 3001
```

This has to stay 80 otherwise it won't work in an exposed setup

```suggestion
      traefik.http.services.vikunja.loadbalancer.server.port: 80
```